components/eamxx/scripts/utils.py

```diff
@@ -228,6 +256,20 @@ def format_time(time_format, input_format, input_time):
         output_time += field[1:]
     return output_time
 
+###############################################################################
+def get_timestamp(timestamp_format="%Y%m%d_%H%M%S", utc_time=False):
```

Comment:
I was thinking, should we just import some of the basic stuff (expect, run_cmd, run_cmd_no_fail, get_timestamp,...) from CIME/utils.py?

Reply:
We could, but that would make us dependent on CIME whereas most of our current scripts are independent.

Reply:
Yeah, that's true, but I was thinking we've been basically copy+pasting CIME utils... Also, CIME should always be available, no?

Tbc, that's not for now, we can do this later on, if we decide to. I was just raising the question.